p2p/simulations/network.go

```diff
@@ -408,6 +404,18 @@ func (net *Network) GetNodes() (nodes []*Node) {
 	return nodes
 }
 
+// GetNodes returns the existing nodes that are up
+func (net *Network) GetUpNodes() (nodes []*Node) {
```

Comment:
I asked about this method a few weeks ago, and it is still not used. I think we should not be committing dead code.